.github/workflows/pytest.yml

```diff
@@ -0,0 +1,192 @@
+name: Reusable WF - Run Pytest
+
+on:
+  workflow_call:
+    inputs:
+      platform:
+        required: true
+        type: string
+      platform-pretty:
+        required: true
+        type: string
+      config-fp:
+        required: false
+        type: string
+      config-pretty:
+        required: false
+        type: string
+        default: "[no config]"
+      python-version:
+        required: true
+        type: string
+      upstream-details:
+        required: false
+        type: string
+      workers:
+        required: false
+        type: string
+        default: "16"
```

Comment:
I would remove this input and just use "-n auto" when invoking, unless you have calling sites that set this explicitly?